h/util/uri.py

```diff
@@ -80,28 +80,28 @@
 # strip from URLs as part of normalization.
 BLACKLISTED_QUERY_PARAMS = [
     re.compile(regex)
-    for regex in {
+    for regex in (
         # Google AdWords tracking identifier. Reference:
         #
         #    https://support.google.com/analytics/answer/2938246?hl=en
         #
         r"^gclid$",
         # Google Analytics campaigns. Reference:
         #
-        #     https://support.google.com/analytics/answer/1033867?hl=en
+        #    https://support.google.com/analytics/answer/1033867?hl=en
         #
         r"^utm_(campaign|content|medium|source|term)$",
         # WebTrends Analytics query params. Reference:
         #
-        #     http://help.webtrends.com/en/analytics10/#qpr_about.html
+        #    http://help.webtrends.com/en/analytics10/#qpr_about.html
         #
         r"^WT\..+$",
         # Amazon security access token. Reference:
         #
-        #     https://docs.aws.amazon.com/general/latest/gr/sigv4-add-signature-to-request.html
+        #    https://docs.aws.amazon.com/general/latest/gr/sigv4-add-signature-to-request.html
         #
         r"(?i)^x-amz-security-token$",
-    }
+    )
```

Comment:
Not sure why this was a set. This just causes unnecessary hashing of the strings.